scripts/build_utils.sh

```diff
@@ -592,6 +597,8 @@ use_ppa() {
                         use_ppa=false
                     fi
                     ;;
+                focal)
+                    use_ppa=true;;
```

Comment:
@tchaikov before this merges, I was hoping you could take a peek at all the changes in this file.  I'm not sure what GCC version should be used or what ppa is, etc.

Reply:
> @tchaikov before this merges, I was hoping you could take a peek at all the changes in this file. I'm not sure what GCC version should be used or what ppa is, etc.

ubuntu focal comes with GCC-9.3, which would suffice for building ceph and crimson.

> 
> I manually pushed the job config changes to the `ceph-dev` job and built Octopus for Ubuntu 20.04 (Focal Fossa). Looks like maybe GCC 10 was used?

```
+ use_ppa=true
...
-- The CXX compiler identification is GNU 9.3.0
-- The C compiler identification is GNU 9.3.0
```

some gcc-10 runtime was downloaded. because the ubuntu-toolchain-r PPA was added somehow.